lib/node_modules/@stdlib/math/base/special/rad2degf/test/test.js

```diff
@@ -0,0 +1,165 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var PINF = require( '@stdlib/constants/float32/pinf' );
+var NINF = require( '@stdlib/constants/float32/ninf' );
+var PI = require( '@stdlib/constants/float32/pi' );
+var EPS = require( '@stdlib/constants/float32/eps' );
+var isnanf = require( '@stdlib/math/base/assert/is-nanf' );
+var abs = require( '@stdlib/math/base/special/abs' );
+var float64ToFloat32 = require('@stdlib/number/float64/base/to-float32');
+var rad2degf = require( './../lib' );
+
+
+// FIXTURES //
+
+var data = require( './fixtures/julia/data.json' );
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.strictEqual( typeof rad2degf, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'if provided `+infinity`, the function returns `+infinity`', function test( t ) {
+	var r = rad2degf( PINF );
+	t.equal( r, PINF, 'returns +infinity' );
+	t.end();
+});
+
+tape( 'if provided `-infinity`, the function returns `-infinity`', function test( t ) {
+	var r = rad2degf( NINF );
+	t.equal( r, NINF, 'returns -infinity' );
+	t.end();
+});
+
+tape( 'if provided `NaN`, the function returns `NaN`', function test( t ) {
+	var r = rad2degf( NaN );
+	t.equal( isnanf( r ), true, 'returns expected value' );
+	t.end();
+});
+
+tape( 'the function converts an angle from radians to degrees', function test( t ) {
+	var expected;
+	var delta;
+	var tol;
+	var x;
+	var r;
+	var i;
+	var e;
+
+	x = data.x;
+	expected = data.expected;
+
+	for ( i = 0; i < x.length; i++ ) {
+		r = rad2degf( x[ i ] );
+		e = float64ToFloat32( expected[ i ] );
+		if ( r === e ) {
+			t.equal( r, e, 'x: '+x[ i ]+'. E: '+e );
+		} else {
+			delta = abs( r - e );
+			tol = EPS * abs( e );
+			t.ok( delta <= tol, 'within tolerance. x: '+x[ i ]+'. y: '+r+'. E: '+e+'. tol: '+tol+'. Δ: '+delta+'.' );
+		}
+	}
+	t.end();
+});
+
+tape( 'the function converts an angle from radians to degrees (canonical values)', function test( t ) {
+	var expected;
+	var delta;
+	var tol;
+	var er;
+	var x;
+	var r;
+	var i;
+
+	x = [
+		PI / 6.0,
```

Comment:
One thing that I wanted to ask here is that, should we use the `float64ToFloat32` wrapper for these values, as they involve arithmetic operations ?